src/main/java/frc/robot/subsystems/intake/PassToScorer.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.intake;
+
+import static frc.robot.Constants.IntakeConstants.kConveyorSpeed;
+import static frc.robot.Constants.IntakeConstants.kIntakeSpeed;
+import static frc.robot.Constants.IntakeConstants.kPivotRetractAngle;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.IntakeConstants.CoralEnum;
+import frc.robot.subsystems.CoralScorer;
+import java.util.function.BooleanSupplier;
+import java.util.function.Supplier;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class PassToScorer extends Command {
+
+  private IntakeSubsystem intakeSubsystem;
+  private BooleanSupplier elevatorNotL1;
+  private CoralScorer coralScorer;
+  private Supplier<CoralEnum> state;
+
+  /** Creates a new PassToScorer. */
+  public PassToScorer(
+      IntakeSubsystem subsystem,
+      BooleanSupplier elevatorNotL1,
+      CoralScorer coralScorer,
+      Supplier<CoralEnum> state) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem);
+    this.intakeSubsystem = subsystem;
+    this.elevatorNotL1 = elevatorNotL1;
+    this.coralScorer = coralScorer;
+    this.state = state;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    intakeSubsystem.goToPivotPosition(kPivotRetractAngle);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if (elevatorNotL1.getAsBoolean() && intakeSubsystem.atSetpoint(kPivotRetractAngle)) {
+      intakeSubsystem.setIntakeMotor(kIntakeSpeed);
+      intakeSubsystem.setConveyerMotor(kConveyorSpeed);
+      coralScorer.scoreCommand().initialize();
+    } else {
+      intakeSubsystem.goToPivotPosition(kPivotRetractAngle);
+      intakeSubsystem.setIntakeMotor(kIntakeSpeed / 5);
+      intakeSubsystem.setConveyerMotor(kConveyorSpeed);
```

Comment:
This case should just be holding the Coral, right? So the motors shouldn't be running